README.md

```diff
@@ -14,7 +14,7 @@ Extract information about the dependencies being updated by a Dependabot-generat
 
 ## Usage instructions
 
-Create a workflow file that contains a step that uses: `dependabot/fetch-metadata@v1`, e.g.
+Create a workflow file that contains a step that uses: `dependabot/fetch-metadata@v1.3.4`, e.g.
```

Comment:
Hmm... I changed this to only pin to major versions here: https://github.com/dependabot/fetch-metadata/pull/246

I didn't realize this was auto-generated, looks like I should update the release script as well??

Reply:
@Nishnha do you want to put up a separate PR modifying the release script per our conversation this morning? I'm guessing you already have a locally modified copy given taht this PR is now fixed 😀

Reply:
I put up https://github.com/dependabot/fetch-metadata/pull/280 to handle this